### ISSUE TITLE
text_personality needs API option

### ISSUE THREAD

**Issue by nbjornberg** [ColumbusCollaboratory/cognizer#15]
API keys are provided now from IBM instead of username:password.

**Comment by tjdrake719** [comment]
Agree.

**Comment by petegordon** [comment]
Yep. We haven't had time to work on this; definitely open to pull requests--if you're able to give us a hand. Thanks.

**Comment by tjdrake719** [comment]
My skillset is more statistics than programming, but if i can figure it out i will submit a pull request.
